api/res/openapi.yml

```diff
@@ -185,6 +221,36 @@ paths:
             text/plain:
               schema:
                 type: string
+  /tasks:
```

Comment:
That's right, also the description of the `/tasks/{taskId}` responses does not exactly match the implementation yet, but that's partly because of the "known issues" mentioned above (polling completed tasks returns 404).

Reply:
Done 👍